### ISSUE TITLE
Gerald sometimes loses ground on the x-axis

### ISSUE THREAD

**Issue by amcewen** [DoESLiverpool/somebody-should#778]
Zombie Chris was seeing problems yesterday with his cuts ending in a different place to their start.  It seems to be only in the x-axis, and wasn't doing it at all today.

So it could be a weird problem with his specific files, but we can leave this issue open for a while to collect any other instances of it happening.

**Comment by amcewen** [comment]
The problem files are saved on the desktop on Gerald's PC, in a folder `Issue Test Files\#778 - drift issue`

**Comment by magman2112** [comment]
I did some testing during Maker night last night and there is an obvious step in the x-axis movement when moving the head with the power switched off.

I found what appear to be the new belts for this axis with @amcewen last night and will endeavour to fit this new belt on Friday to see if this fixes the issue. I suspect that there may also be some belt alignment which will also be fixed at the same time.
